docs/nodes/validator-node-handling.md

```diff
@@ -104,6 +104,140 @@ The validator tab should have the value **Operator**. That value is your `<desti
 
 If you are using the **Ledger device** you must first connect it to your computer, start the cosmos application and add `--ledger` flag to the command.
 
+
+## Move node to other server
+
+If you need move your validator to another server only thing that you need is move your private key.     
```

Comment:
_If you need to move your validator to another server, the only thing that you need to move is your private key_